src/main/java/net/snowflake/ingest/streaming/internal/SnowflakeStreamingIngestClientInternal.java

```diff
@@ -167,6 +171,8 @@
       SharedMetricRegistries.add("Metrics", metrics);
     }
 
+    this.parameterProvider.setParameterMap(parameterOverrides, prop);
```

Comment:
Could we add a log after this so we could know the actual values being used in the end?